src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -228,9 +228,8 @@ def load_arguments(self, _):
                    help='The type of the web application firewall rule set.')
         c.argument('rule_set_version',
                    options_list='--version',
-                   arg_type=get_enum_type(['0.1', '2.2.9', '3.0', '3.1', '3.2']),
-                   help='The version of the web application firewall rule set type. '
-                        '0.1 is used for Microsoft_BotManagerRuleSet')
+                   arg_type=get_enum_type(['0.1', '1.0', '2.2.9', '3.0', '3.1', '3.2']),
```

Comment:
Can we define this enum value as a constant and reuse it in multiple places?

Reply:
I mean if we can set something like {"0.1": "0.1", "1.0": "1.0", "2.2.9": "2.2.9", "3.0": "3.0", "3.1": "3.1", "3.2": "3.2"} to a constant and reuse it in multiple places, so we only need update the constant in the future if we have new versions; otherwise we still need update 2 places.